_posts/2018-04-25-beta-release.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Pulp 3.0 Core Beta Release
+author: Austin Macdonald
+tags:
+  - release
+  - 3.0
+---
+
+## pulpcore 3.0 beta
+
+Pulpcore 3.0 beta 1 has been released, and can be [installed from
+PyPI](https://docs.pulpproject.org/en/3.0/nightly/installation/instructions.html).
+
+[Pulp 3 documentation](https://docs.pulpproject.org/en/3.0/nightly/index.html) is built nightly.
+Pulp 3 concepts and changes are covered in our [overview documentation](https://docs.pulpproject.org/en/3.0/nightly/overview/index.html).
+If you find problems or have features you’d like to request please [file an issue](https://docs.pulpproject.org/en/3.0/nightly/bugs-features.html)
+
+
+## Pulp 3 Highlighted New Features
+
+- Relational database. We designed Pulp 3 to be SQL agnostic, it was developed with SQLite and
+  PostgreSQL.
+- Pulp 3 is installable with pip, and can be run on most linux systems.
+- The set of content in each repository is versioned.
+- Promotion and rollback are fast, synchronous calls.
+- Auto-generated, always up-to-date [API reference
+  documentation](https://docs.pulpproject.org/en/3.0/nightly/integration-guide/rest-api/index.html#pulpcore-rest-api)
+
+
+## What to expect
+
+#### Weekly Releases to PyPI
+
+Releases will continue to be announced on the lists.
```

Comment:
Can we link to our two packages on PyPI somehow?   pulpcore and pulpcore-plugin

https://pypi.org/project/pulpcore/
https://pypi.org/project/pulpcore-plugin/

Reply:
How about "Releases of [pulpcore](link) and [pulpcore-plugin](link) will continue to be announced on the lists.

Reply:
Looks great, ty